### PR TITLE
Fixed naming bugs

### PR DESCRIPTION
**Pull request by rcoleworld**
Fixed spelling of contributors and changed "thumbnailUrl" to "thumbnail".

### CODE REVIEW

File: anchor-api/lib/models/article.model.ts [hunk 1]
```diff
@@ -10,32 +10,32 @@ import AIOutput from './AIOutput';
...
-  firstPublishDate: Date,
-  lastPublishDate: Date,
-  contributers: string [],
-  headline: string,
-  section: string,
-  thumbnailUrl: string,
-  body: string | AIOutput[],
-  category: string, // TODO make this an enum
-  publisher: string
+  url: string;
```
Comment (iCePU) [inline anchor-api/lib/models/article.model.ts:13]: Ideally we need to go into this branch and revert these white space changes but eh

File: anchor-api/lib/models/article.model.ts [hunk 2]
```diff
@@ -10,32 +10,32 @@ import AIOutput from './AIOutput';
...
+  headline: string;
+  section: string;
+  thumbnail: string;
+  body: string | AIOutput[];
+  category: string; // TODO make this an enum
+  publisher: string;
 }
 
 const ArticleSchema: Schema = new Schema({
-  url: { type: String, required: true , unique: true},
```
Comment (iCePU) [inline anchor-api/lib/models/article.model.ts:26]: same here

File: anchor-api/lib/models/article.model.ts [hunk 3]
```diff
@@ -10,32 +10,32 @@ import AIOutput from './AIOutput';
...
+  url: { type: String, required: true, unique: true },
```
Comment (iCePU) [inline anchor-api/lib/models/article.model.ts:26]: and here

**Review by iCePU — APPROVED** [review]
looks good just need to get in the habit of not messing with white space between commits. Dirties up the repo and makes it hard to read the diffs

**Comment by nxtHiro** [comment]
LGTM

**Pull request merged**